sdk/formrecognizer/ai-form-recognizer/CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Release History
 
+## 4.1.0-beta.1 (2023-04-10)
+
+### Features Added
+
+- Added support for Form Recognizer API version `2023-02-28-preview`. This is the default API version for SDK version `4.1.0-beta.1`. For a detailed list of changes included in this API version, see the [Form Recognizer release notes](https://learn.microsoft.com/azure/applied-ai-services/form-recognizer/whats-new?view=form-recog-3.0.0&tabs=javascript#march-2023).
+- Added support for custom classification models. Classifiers combine layout and language features to accurately identify documents from among a set of document types.
+  - Added `beginBuildDocumentClassifier` to `DocumentModelAdministrationClient` to build a custom classifier from training documents. See [the service documentation on custom classifiers](https://aka.ms/azsdk/formrecognizer/buildclassifiermodel) for more information on creating a training data set and creating a classifier.
+  - Added `getDocumentClassifier`, `listDocumentClassifiers`, and `deleteDocumentClassifiers` methods to `DocumentModelAdministrationClient` to manage custom classifiers.
+  - Added `beginClassifyDocument` and `beginClassifyDocumentFromUrl` to `DocumentAnalysisClient` to classify documents using a custom classification model.
+- Added `features` to `AnalyzeDocumentOptions`. Features are add-on capabilities that must be explicitly activated in order to extract them. The use of features may incur additional costs, so please review the [service documentation of add-on capabilities](https://aka.ms/azsdk/formrecognizer/features) for more information. The Form Recognizer service has introduced four features in this API version:
+  - `ocr.highResolution`: improves the quality of content extraction for A1/A2/A3 documents with small text.
+  - `ocr.formula`: enables extraction of formulas, such as mathematical equations.
+  - `ocr.font`: extends the existting `styles` property to include more font properties such as `similarFontFamily` for the font face of the text, `fontStyle` for styles such as italics, `fontWeight` for boldness, `color`, and `backgroundColor`.
+  - `queryFields.premium`: enables the use of `queryFields` (see below).
+- Added `queryFields` to `AnalyzeDocumentOptions`. The service now supports extracting additional query fields using Azure OpenAI capabilities. This feature may only be used if the `"queryFields.premium"` feature is enabled (see above). See [the service documentation on query fields](https://aka.ms/azsdk/formrecognizer/queryfields) for more information.
+
```

Comment:
will you mention other features added, like the new properties on AddressValue, DocumentPage, CurrencyValue, DocumentField, etc?

Reply:
Yep, changelog is now finalized.